src/flexbox/nasa.html

```diff
@@ -0,0 +1,52 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>NASA page</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./nasa.css">
+</head>
+<body>
+    <div class="main-container">
+        <div class="nasa">
+            <h2>NASA</h2>
+        </div>
+        <footer>
+            <div class="social">
+                    <ul>
+                        <h3>Social Media</h3>
+                        <li>Twitter</li>
+                        <li>Facebook</li>
+                        <li>Instagram</li>
+                        <li>Youtube</li>
+                    </ul>
+                    <ul>
+                        <h3>OFFICE INFO</h3>
+                        <li>Office Comms</li>
+                        <li>NASA Headquaters</li>
+                        <li>300 E Street</li>
+                        <li>Austin, Tx</li>
+                        <li>(512) 459-2222</li>
+                    </ul>
+                    <form>
+                        <h3>CONTACT US</h3>
+                        Name<br>
+                        <input type="text" id="name"><br>
+                        Message<br>
+                        <input id="message"><br>
+                        
+                        <button type="button" id="button">SEND</button>
+                    </form>
+                    
+
+            
+            </div>
+
+        </footer>
+
+
+    </div>
+    
+</body>
+</html>
```

Comment:
There's a lot of extra space. clean it up. keep it tight. also, did you set vscode tab size to 2? go to preferences => settings => search for 'tab size' and set to '2'. you may have to restart vscode to get it set!